cartridges/int_klaviyo_core/cartridge/scripts/klaviyo/utils.js

```diff
@@ -222,6 +222,98 @@ function trackEvent(exchangeID, data, event, customerEmail) {
 }
 
 
+// The subscribeUser func takes the user email & phone number to prep a data object w/ a corresponding emailListID or smsListID (both configured in BM w/ values from the Klaviyo Dashboard)
+// Data is sent to the KlaviyoSubscribeProfilesService API to subscribe users to email or SMS lists.
+function subscribeUser(email, phone) {
+    var logger = Logger.getLogger('Klaviyo', 'Klaviyo.core utils.js - subscribeUser()');
+
+    if (klaviyoServices.KlaviyoSubscribeProfilesService == null) {
+        logger.error('subscribeUser() failed - KlaviyoSubscribeProfilesService is null.');
+        return;
+    }
+
+    var emailListID = Site.getCurrent().getCustomPreferenceValue('klaviyo_email_list_id');
+    var smsListID = Site.getCurrent().getCustomPreferenceValue('klaviyo_sms_list_id');
+
+    var data;
+    var result;
+
+    if (smsListID === emailListID && email && phone) {
```

Comment:
See the slack thread [here](https://klaviyo.slack.com/archives/C04S7314ELF/p1688072491921639) for more info but we will need to always include both `email` and `phone_number` on the subscribe call regardless of which channel is being subscribed to in order to create unified profiles in klaviyo. Effectively we can achieve this by adjusting the arguments being passed to this method (more in thread) and removing this first conditional.